patches/icu_suffix_only_on_libname.patch

```diff
@@ -0,0 +1,24 @@
+--- icu/source/configure	2013-05-23 20:06:32.000000000 +0000
++++ icu/source/configure.patched	2018-02-14 00:00:15.264160824 +0000
+@@ -7217,14 +7217,14 @@
+ { $as_echo "$as_me:${as_lineno-$LINENO}: result: $msg" >&5
+ $as_echo "$msg" >&6; }
+ 
+-if test "$ICULIBSUFFIX" != ""
+-then
+-    U_HAVE_LIB_SUFFIX=1
+-    ICULIBSUFFIXCNAME=`echo _$ICULIBSUFFIX | sed 's/^A-Za-z0-9_/_/g'`
+-    UCONFIG_CPPFLAGS="${UCONFIG_CPPFLAGS} -DU_HAVE_LIB_SUFFIX=1 -DU_LIB_SUFFIX_C_NAME=${ICULIBSUFFIXCNAME} "
+-else
++#if test "$ICULIBSUFFIX" != ""
++#then
++#    U_HAVE_LIB_SUFFIX=1
```

Comment:
given that this is a temporary fix anyway, maybe it'd be better to just remove this section in the patch rather than comment it out?

Reply:
agreed, but other than that this looks good to me

Reply:
sure! I'll get it updated today.